styles.css

```diff
@@ -0,0 +1,104 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500;600;700&display=swap");
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  max-width: 375px;
+  font-family: "Poppins", sans-serif;
+  margin: 0 auto;
+  background: #f7f7f9;
+}
+
+header {
+  background-color: #fff;
+  max-width: 375px;
+  height: 20px;
+  padding-top: 28px;
+}
+
+.container1 {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  color: #6070ff;
+  margin-top: 58px;
+}
```

Comment:
> Use responsive values in your CSS rules, like percentages (i.e. width: 80%), instead of pixels

- [ ] Looking at the project requirements, it suggests using responsive values for widths instead of pixels. Looking at your CSS, you tend to use a lot of pixel values for your max-widths, I'll suggest using percentages, it makes the website easily responsive and adapts well to different mobile screen sizes. :+1: :+1: 
- [ ] Looking at your design, you have a margin-top that is not in the Figma design, I'll suggest removing this extra margin-top as it creates unnecessary space at the top of your portfolio. :+1: :smile: 

- - Screenshot:
![Selection_060](https://github.com/sadat-Ramin/My-portfolio/assets/107632780/303b1ca0-7d7a-48d4-ad35-47cda5d87c96)